assets/book/typography/styles/_NotoNastaliqUrduFont.scss

```diff
@@ -0,0 +1,15 @@
+// 'Noto Nastaliq Urdu', @import 'NotoNastaliqUrduFont'
+
+@font-face {
+  font-family: 'Noto Nastaliq Urdu';
+  font-style: normal;
+  font-weight: normal;
+  src: url('uploads/assets/fonts/NotoNastaliqUrdu-Regular.otf') format('opentype');
+}
+
+@font-face {
+  font-family: 'Noto Nastaliq Urdu';
+  font-style: normal;
+  font-weight: bold;
+  src: url('uploads/assets/fonts/NotoNastaliqUrdu-Bold.otf') format('opentype');
```

Comment:
I think according to the conventions the path has to be only `fonts/NotoNastaliqUrdu-Bold.otf` if not it will trigger an error when saving the theme options.

For instance:

<img width="591" alt="image" src="https://github.com/user-attachments/assets/83ce2f23-1872-4f4e-9800-dd8131ed4847" />